mpqp/core/instruction/measurement/basis_measure.py

```diff
@@ -49,12 +50,14 @@ class BasisMeasure(Measure):
 
     def __init__(
         self,
-        targets: list[int],
+        targets: Optional[list[int]] = None,
         c_targets: Optional[list[int]] = None,
-        shots: int = 0,
+        shots: int = 1024,
```

Comment:
I think should discuss it before really doing it,it was just a proposition. @Henri-ColibrITD  @MoHermes look at the comment line 53